Example/StatusViewController.swift

```diff
@@ -6,11 +6,20 @@
 @testable import SchibstedAccount
 import UIKit
 
-extension StatusViewController: IdentityManagerDelegate {
-    // IdentityManagerDelegate
-    func userStateChanged(_: UserState) {
-        print("Login state changed to: \(self.isUserLoggedIn)")
-        self.updateFromCurrentUser()
+extension StatusViewController: UserDelegate {
+    func user(_ user: User, didChangeStateTo state: UserState) {
+        print("user \(user) changed state to \(state)")
+        switch state {
+        case .loggedIn:
+            // This can only happen with headless login because a visual login results in IdentityUIDelegate.didFinish
+            UIApplication.currentUser = UIApplication.identityManager.currentUser
```

Comment:
Why do you have two currentUsers? Is it because of a delegate issue?

Reply:
Because the example app uses two login methods which official is not recommended. I.e. apps should only use the UI to login, but the example app is also a playground to display all the features so it uses the headless as well. So we have two `IdentityManager` objects around.